dlpy/tests/test_network.py

```diff
@@ -97,21 +97,6 @@ def test_inputs(self):
             output1 = OutputLayer(2)(conv1)
             Model(conn=self.s, inputs=[conv1], outputs=[output1])
 
-    def test_outputs(self):
```

Comment:
this should be linked with removing that if-else check for can_be_last_layer, right?

Reply:
yes.